packages/open-next/src/build.ts

```diff
@@ -489,6 +489,25 @@ async function createServerBundle(monorepoRoot: string) {
   injectMiddlewareGeolocation(outputPath, packagePath);
   removeCachedPages(outputPath, packagePath);
   addCacheHandler(outputPath);
+
+  if (options.minify) {
+    removeNodeModule(path.join(outputPath, "node_modules"), [
+      "@esbuild",
+      "prisma/libquery_engine-darwin-arm64.dylib.node",
+      "@swc/core-darwin-arm64",
+      "@swc/core",
+      "better-sqlite3",
+      "esbuild",
+      "webpack",
+      "uglify-js",
+      // "react", // TODO: remove react/react-dom when nextjs updates its precompile versions
+      // "react-dom",
+      "@webassemblyjs",
+      "uglify-js",
```

Comment:
Duplicate here, it's also listed above!

Reply:
Thx, I'll have to retest this w/ Next 14, which seems to have fixed the dev deps getting traced to the standalone output.

Reply:
Been busy last week and caught a cold this week... I'll eventually test this to confirm... @mathisobadia was still seeing these dev modules but I didn't...

Reply:
Here's my server function lambda zip, on Next 14.0.2:

![image](https://github.com/sst/open-next/assets/245131/b146244e-1043-4f95-a781-ebed5644a4d4)

Reply:
@revmischa This is inside your server-function lambda ? There is a lot of things that you should be able to remove, you can use `experimental.outputFileTracingExcludes` from next config to remove all of those. 
You should probably create an issue in next repo as well, those dev deps should have been removed.